### PR TITLE
Rename "size" field to "capacity" in Room 

### LINKED ISSUE #1625: Rename "size" field to "capacity" in Room
Venue rooms have a size field which seems to be the number of attendees it can accommodate. This should be named capacity as size is an ambiguous term and probably not the first one would think of for that.

### PR DESCRIPTION
**Pull request by rahul2240**
Renamed "size" field to "capacity" in room
fixes #1625

### CODE REVIEW

File: app/views/admin/rooms/_form.html.haml [hunk 1]
```diff
@@ -10,6 +10,6 @@
   .col-md-8
     = semantic_form_for(@room, url: (@room.new_record? ? admin_conference_venue_rooms_path : admin_conference_venue_room_path(@conference.short_title, @room))) do |f|
       = f.input :name, input_html: { autofocus: true}
-      = f.input :size, input_html: {size: 5}
+      = f.input :size,label: 'Capacity', input_html: {size: 5}
```
Comment (Ana06) [inline app/views/admin/rooms/_form.html.haml]: can you please leave a space between _size_ and _label_, so:

`= f.input :size, label: 'Capacity', input_html: {size: 5}`

just for readability :wink: I wonder why our linters are not complaining :thinking:

**Comment by rahul2240** [comment]
@Ana06  Thanks! I will include screenshot next time.

File: app/views/admin/rooms/_form.html.haml (hunk 1 above)
Comment (rahul2240) [inline app/views/admin/rooms/_form.html.haml]: @Ana06 I did :smile:

**Comment by differentreality** [comment]
@rahul2240 nice :+1: Thank you :tada:

**Pull request merged**